pom.xml

```diff
@@ -280,6 +280,31 @@
                 <module>aggregator</module>
             </modules>
         </profile>
+        <profile>
+            <id>release324</id>
+            <activation>
+                <property>
+                    <name>buildver</name>
+                    <value>324</value>
```

Comment:
this needs to be included into `all.buildvers`

Reply:
I think `all.buildvers` is defined to include `noSnapshot.buildvers`, correct?

```
        <all.buildvers>
            ${noSnapshot.buildvers},
            ${snapshot.buildvers},
            ${databricks.buildvers},
        </all.buildvers>
```

Reply:
Yes, it does https://github.com/NVIDIA/spark-rapids/blob/ea89dd58cb101834703da4e3b2b807cd4f0fbb6f/pom.xml#L706

Reply:
it does but I overlooked the line https://github.com/NVIDIA/spark-rapids/pull/8239/commits/d4e7063abb44d9381e20f85901292b47411f118d#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R699